cmd/publishing-bot/publisher.go

```diff
@@ -65,7 +65,7 @@ func (p *PublisherMunger) updateSourceRepo() (string, error) {
 		return "", fmt.Errorf("failed running %v on %q repo: %v", strings.Join(cmd.Args, " "), p.config.SourceRepo, err)
 	}
 
-	rules, err := config.LoadRules(p.config.RulesFile)
+	rules, err := config.LoadRules(p.config.RulesFile, sourceBranch, repoDir)
```

Comment:
can we have the checkout here and remove the branch handling for LoadRules?

Reply:
i dunno what consequences the checkout can have here (i guess we will then checkout to rule branches anyway?)